ui/app/components/mount-backend-form.js

```diff
@@ -86,112 +80,113 @@ export default Component.extend({
     if (!currentPath || isUnchanged) {
       mount.set('path', type);
```

Comment:
since it's glimmerized, can we do `mount.path = type` here?

Reply:
Yes, let me check that out.